.github/workflows/trigger-metamask-institutional-e2e-ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Trigger MetaMask Institutional E2E CI
+
+on:
+  push:
+    branches: [ Version-v*, master ]
+  pull_request:
+    branches: [ Version-v*, master ]
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  trigger-mmi-e2e-ci:
+    runs-on: ubuntu-latest
+    if: ${{ (!github.event.pull_request.draft && (startsWith(github.head_ref, 'Version-v') || github.head_ref == 'master')) || (!github.event.pull_request) }}
+    steps:
+      - name: Trigger MetaMask Institutional E2E CI
+        run: |
+          curl -L -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ secrets.MMI_E2E_CI_TOKEN }}" \
```

Comment:
Secrets aren't accessible in this manner; they need to be passed in using environment variables

See https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#example-using-bash
Also see the `trigger-metamask-desktop-ci.yml` workflow in this repository for another example

Reply:
That was weird. I tested it on the [mmi e2e repo](https://github.com/consensys-vertical-apps/mmi-extension-e2e/pull/17/checks) and worked... maybe it was due to run the curl in the same repo.. anyway, it is updated now